src/MicrosoftTeams.ts

```diff
@@ -861,6 +856,21 @@ export function showNotification(
   sendMessageRequest(parentWindow, "showNotification", params);
 }
 
+/**
+ * @private
+ * Hide from docs.
```

Comment:
Why mark this as @private? I think this is an API we actually want made available to all apps.

Reply:
I am planning to keep it private till this is available in teams Public Ring